### ISSUE TITLE
Using setLightValue cases sides not to be rendered

### ISSUE THREAD

**Issue by Silly511** [Chisel-Team/CTMLib#5]
I don't really know how to explain it. I have two CTM blocks(one transparent the other solid) and I made both of them emit light. However when I did, if a side of my solid block is up against a side of my transparent block then that side does not render. But it only does this when its my transparent block, not with things like glass and stuff. Everything works fine when I remove the light. [Here are some pictures.](http://imgur.com/a/UxIag)

**Comment by tterrag1098** [comment]
do you override isOpaqueCube and isNormalCube ?

**Comment by Silly511** [comment]
Yes, like I said when I take setLightValue away then everything works fine. This only happens when I use setLightValue.

**Comment by tterrag1098** [comment]
Do you mean `setLightLevel` ? I see no reason that would affect anything.

**Comment by Silly511** [comment]
Thats what I thought too but when I take it away it works, when I put it back it does this.

**Comment by tterrag1098** [comment]
Could I see your code for this block, and whatever block that is next to it?

**Comment by Silly511** [comment]
I'm out right now so I can't post my exact code but I can tell you that my block class extends a class I made called connected textures thats exactly like CTMBlock except returns the render id for CTMRenderer. I'll post the actual code as soon as I can.
